examples/posenet/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
I would suggest to remove this file and move the `imageDemo.html` (rename to `index.html`) and `camera.html` code to this folder level (and removing cameraDemo and imageDemo folders). Reuse code as much as possible between the two demos.